openapi/social/social.yaml

```
@@ -0,0 +1,276 @@
+openapi: 3.1.0
+info:
+  title: Social
+  version: '1.0'
+  description: These APIs allow users to use Vendasta's social capabilities. All the endpoints require either the `social` or `business` scopes to be accessed.
+servers:
+  - url: 'https://prod.apigateway.co/product/social'
+    description: Production
+  - url: 'https://demo.apigateway.co/product/social'
+    description: Demo
+  - url: 'http://localhost:3000/product/social'
+    description: Local
+paths:
+  /socialProfiles:
+    get:
+      summary: List Social Profiles
+      tags:
+        - socialProfiles
```

These are displayed to users in the navigation.
```suggestion
        - "Social Profiles"
```